.github/workflows/release.yml

```diff
@@ -143,11 +148,13 @@ jobs:
           mkdir ./npm_publish/dist/go-ios-darwin-amd64_darwin_amd64
           mkdir ./npm_publish/dist/go-ios-darwin-arm64_darwin_arm64
           mkdir ./npm_publish/dist/go-ios-linux-amd64_linux_amd64
+          mkdir ./npm_publish/dist/go-ios-linux-arm64_linux_arm64
           mkdir ./npm_publish/dist/go-ios-windows-amd64_windows_amd64
           cp ./mac-bin/ios ./npm_publish/dist/go-ios-darwin-amd64_darwin_amd64/ios
           cp ./mac-bin/ios ./npm_publish/dist/go-ios-darwin-arm64_darwin_arm64/ios
           cp ./win-bin/ios.exe ./npm_publish/dist/go-ios-windows-amd64_windows_amd64/ios.exe
           cp ./bin/ios ./npm_publish/dist/go-ios-linux-amd64_linux_amd64/ios
```

Comment:
can't we use here the platform specific ones: `bin/ios-amd64 bin/ios-arm64`

Reply:
Yes but I didn't want to do a big change. Using the fat binary for both platforms is the original implementation for macOS, so just followed that.